spec/models/concerns/courses/edit_options_spec.rb

```diff
@@ -4,6 +4,21 @@
   let(:course) { create(:course, level: "primary", subjects: [subjects]) }
   let(:subjects) { create(:subject, :primary) }
 
+  context "subjects" do
```

Comment:
I know that this may feel like knit-picking, so apologies if it feels like that, but I think we should be standardising on how we use `describe` and `context`. For the sake of readability, because clearly functionally they are the same. As it turns out, this article reflects my own thinking on how these are used: https://lmws.net/describe-vs-context-in-rspec

Essentially, `describe` is for what functionality is being tested and `context` is the state the system is in for a particular test or set of tests. We should probably document this in the [spec README](https://github.com/DFE-Digital/manage-courses-backend/blob/master/spec/README.md).

To wit, this here feels like it should be a `describe` block, since we're testing the subjects functionality of `edit_options` here.

Reply:
Fair enough, I just forgot